ci/assets/template/director-config.yml

```diff
@@ -70,3 +70,11 @@ resource-configuration:
   compilation:
     instance_type:
       id: automatic
+{{if index . "pks_lb_network_sg"}}
+vmextensions-configuration:
+  - name: pks-api-lb-security-groups
+    cloud_properties:
+      security_groups: 
+      - {{.pks_lb_network_sg}}
```

Comment:
https://bosh.io/docs/azure-cpi/#resource-pools

[If I'm reading this right](https://bosh.io/docs/azure-cpi/#resource-pools) then `security_groups` isn't a cloud property, it's just `security_group` which takes in a single string. Should this be `application_security_groups` and have the pks lb ASG associated to it (and drop the default sg)?

am currently testing this

Reply:
I apologize, I haven't revisited this since I made the initial commit, I definitely ended up doing some tweaking in the UI to get it to work, so I don't believe this is a full solution.

Reply:
I gotchu no worries all good. Do you remember anything else you had to change, by any chance? Currently having some trouble getting the security groups on the PKS VM/LB associated correctly, though I'm wondering if you recall anything else different you had to do or snags you ran into.

No worries if you don't remember or don't have time to respond, am just curious.

Reply:
for reference to anyone else seeing this, what I did to get this to work was _assuming you're deploying the PKS API VM in the Infra subnet_, I had to specify the `ops-manager-nsg` as the `security_group:`, and `pks-api-asg` as a member of `application_security_groups:`

```yml
vmextensions-configuration:
  - name: pks-api-lb-security-groups
    cloud_properties:
      security_group: ops-manager-nsg
      application_security_groups: 
      - pks_lb_asg
```